state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -18,21 +19,182 @@
 import pwmio
 import bmx160
 import drv8830
-from os import listdir, stat, statvfs, mkdir
 from bitflags import bitFlag, multiBitFlag
 from micropython import const
 import adafruit_tsl2561
 
+
+"""
+IMU-related functions
+"""
+
+def acceleration():
+    """ return the accelerometer reading from the IMU """
+    return _cubesat.IMU.accel
+
+def magnetic():
+    """ return the magnetometer reading from the IMU """
+    return _cubesat.IMU.mag
+
+def gyro():
+    """ return the gyroscope reading from the IMU """
+    return _cubesat.IMU.gyro
+
+def temperature_imu():
+    """ return the thermometer reading from the IMU """
+    return _cubesat.IMU.temperature  # Celsius
+
+
+"""
+Burnwire-related functions
+"""
+
+def burn(burn_num='1', dutycycle=0, freq=1000, duration=1):
+    """
+    control the burnwire(s)
```

Comment:
I don't understand what the function does, if you do could you please explain it to me in the docstring.

Reply:
Yep, will do